### ISSUE TITLE
DEPRECATION: Addon files were detected in `node_modules/ember-cli-stripe/addon`, but no JavaScript preprocessors were found 

### ISSUE THREAD

**Issue by msalahz** [sweettooth/ember-cli-stripe#24]
I'm getting this warning after ember s 
```
DEPRECATION: Addon files were detected in `/node_modules/ember-cli-stripe/addon`, but no JavaScript preprocessors were found for `ember-cli-stripe`. Please make sure to add a preprocessor (most likely `ember-cli-babel`) to in `dependencies` (NOT `devDependencies`) in `ember-cli-stripe`'s `package.json`.
```

**Comment by vladucu** [comment]
@msalahz what version of ember-cli /  ember are you using?

**Comment by msalahz** [comment]
ember-cli: 2.12.1
node: 6.10.0
os: darwin x64

**Comment by vladucu** [comment]
should be fixed with #26
